lib/new-syntax/pretty.ts

```diff
@@ -274,5 +276,5 @@ export function prettyprint(tokens : TokenStream) : string {
         }
     }
 
-    return buffer.trim();
+    return buffer.replace(/ +/g, ' ').trim();
```

Comment:
In what case do we generate extra spaces?

Reply:
`enum on`

Reply:
but apparently this is bad... we have indents, maybe `(?<!\s) +` would work?

Reply:
but apparently this won't work as we have another space after `on` and there are also indents. 
I updated this to check if the buffer ends with `enum`. Is there any other case when a keyword is not used as a keyword?

Reply:
I think units have special keyword behavior but those are handled differently anyway, so yeah that should be it?